app/views/users/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>List of Users</h1>
\ No newline at end of file
+<ul>
+  <% @users.each do |user| %>
+    <li>
+      <a href="<%= user_posts_path(user) %>">
+        <div class="custom-user-box">
+          <%= render partial: 'user_helper/user_photo', locals: { user: user } %>
+          <%= render partial: 'user_helper/user_details', locals: { user: user } %>
+        </div>
+      </a>
+    </li>
```

Comment:
- [ ] on click of a user card should take me to the user-show page not user-post- index page, kindly fix that